Makefile

```diff
@@ -25,6 +25,10 @@ vendor:
 build:
 	$(DOCKER_RUN) go build -o $(DIST) $(GO_BUILD_ARGS)
 
+.PHONY: test
+test:
+	$(DOCKER_RUN) go test ./pkg/...
```

Comment:
but I actually think yours is better because glide may not be installed with go only lives inside docker